### PR TITLE
Update `CONTRIBUTING.md`

### PR DESCRIPTION
**Pull request by rudychung**
Addresses #32 

I'm sure I missed a few things, let me know if there's anything else you want me to add to the document.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Getting Started
 
-Install Node.js if you haven't already and run:
+Install [Node.js](https://nodejs.org/) if you haven't already, and run to install project dependencies:
```
Comment (batunpc) [inline CONTRIBUTING.md]: Maybe we could rephrase the sentence like 

Install [Node.js](https://nodejs.org/) if you haven't already, and run the following command to install project dependencies:

**Review by batunpc — CHANGES_REQUESTED** [review]
Hi @rudychung, your instructions looking good! I just have found very minor issue and rephrased the sentence. 

Other than that It looks good!

**Review by batunpc — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -9,19 +9,35 @@ To start the local site use:
...
+`npm run lint`
+
 # Using Cloudflare Workers
 
 1. Run `npx wrangler dev <filepath>` in your terminal to start a development server
 1. Open a browser tab at http://localhost:8080/ to see your worker in action
 1. Run `npx wrangler publish <file-path> --name <worker-name>` to publish your worker
 
 Learn more at https://developers.cloudflare.com/workers/
+
```
Comment (SerpentBytes) [inline CONTRIBUTING.md]: Is it possible to restructure the document so that the "How to Contribute" section appears before "Getting Started"?

**Review by SerpentBytes — CHANGES_REQUESTED** [review]
I left some feedback.

**Review by SerpentBytes — APPROVED** [review]

**Pull request closed without merging**